internal/engine/pod.go

```diff
@@ -218,21 +218,34 @@ func getBlessedContainerID(iTargets []model.ImageTarget, pod *v1.Pod) (container
 	}
 	return "", nil
 }
-func checkForPodCrash(ctx context.Context, state *store.EngineState, ms *store.ManifestState, podInfo store.Pod) {
+
+func checkForPodCrash(ctx context.Context, state *store.EngineState, mt *store.ManifestTarget) {
```

Comment:
nit: we're not JUST checking for pod crash now -- it's pod-or-container-crash. idk what i'd rename it to tho

Reply:
changed to checkForContainerCrash